src/point.rs

```diff
@@ -61,6 +68,17 @@ impl<T: NumCast> Point<T> {
     }
 }
 
+
+impl Position for Point<u32> {
+    fn x(&self) -> u32 { self.x }
+    fn y(&self) -> u32 { self.y }
+}
+
+impl Position for Point<usize> {
+    fn x(&self) -> u32 { self.x as u32 }
+    fn y(&self) -> u32 { self.y as u32 }
+}
```

Comment:
It's probably not a great idea to implement this trait using casts, it could cause some very hard to track down bugs. It might be a better idea to do this conversion manually where needed.